benchmark/torch/sac/train.py

```diff
@@ -0,0 +1,150 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import gym
+import argparse
+from parl.utils import logger, tensorboard, ReplayMemory
+from parl.env.continuous_wrappers import ActionMappingWrapper
+from mujoco_model import MujocoModel
+from mujoco_agent import MujocoAgent
+from parl.algorithms import SAC
+
+WARMUP_STEPS = 1e4
+EVAL_EVERY_STEPS = 5e3
+EVAL_EPISODES = 5
+MEMORY_SIZE = int(1e6)
+BATCH_SIZE = 256
+GAMMA = 0.99
+TAU = 0.005
+ALPHA = 0.2  # determines the relative importance of entropy term against the reward
```

Comment:
Put it at the parser. The various benchmarks require different values for this argument.

Reply:
Please notify the user at the readme that Humanoid requires 0.2 while other environments require 0.05 .

Reply:
done